src/combine-reducers.js

```diff
@@ -0,0 +1,21 @@
+'use strict'
+
+module.exports = function combineReducers (reducers) {
+  // ensure array
+  var _reducers
+  if (!Array.isArray(reducers)) {
+    _reducers = [reducers]
+  } else {
+    _reducers = reducers
+  }
+
+  // I just want to say I think this is illegiable, and I think
+  // this code is much better understood in its original form:
+  //
+  // return function (initialState, action) {
```

Comment:
I don't necessarily disagree though I will also note that after reading nested arrow functions more and more they become more and more intuitive.